framework/tests/sideeffectsonly/SideEffectsMultiple.java

```diff
@@ -0,0 +1,26 @@
+package sideeffectsonly;
+
+import org.checkerframework.dataflow.qual.SideEffectsOnly;
+import org.checkerframework.framework.qual.EnsuresQualifier;
+import org.checkerframework.framework.testchecker.sideeffectsonly.qual.Refined;
+
+public class SideEffectsMultiple {
+    void test(Object x) {
+        method(x);
+        method1(x);
+        // :: error: argument.type.incompatible
+        method2(x);
+    }
+
+    @EnsuresQualifier(
+            expression = "#1",
+            qualifier =
+                    org.checkerframework.framework.testchecker.sideeffectsonly.qual.Refined.class)
+    // :: error: contracts.postcondition.not.satisfied
+    void method(Object x) {}
+
+    @SideEffectsOnly({"this", "x"})
+    void method1(@Refined Object x) {}
```

Comment:
Throughout, be sure to add tests where the formal parameter name differs.  As written, the formal parameters are all `x`.